pytorch_lightning/utilities/imports.py

```diff
@@ -53,13 +53,9 @@ def _module_available(module_path: str) -> bool:
     if not _package_available(module_names[0]):
         return False
     try:
-        module = importlib.import_module(module_names[0])
-    except ImportError:
+        importlib.import_module(module_path)
+    except ModuleNotFoundError:
         return False
-    for name in module_names[1:]:
```

Comment:
looking at git blame, this came from #11526 which on the other hand was carried over from a report in torchmetrics: https://github.com/PyTorchLightning/metrics/issues/770

So, to verify that @JoostvDoorn's fix is valid, we can check against that original issue to make sure its' covered :)

Reply:
cc @Borda as the author of this piece of code

Reply:
cc @Borda

Reply:
yep, did test adjustments and seems to be passing